simulator/setting_up_simulator.md

```diff
@@ -42,7 +59,32 @@ pip install -r competition-simulator-<version>/libraries.txt
 ~~~~~
 
 
-## Installing the simulation
+### Troubleshooting: setting your version of Python
+
+Sometimes Webots will not automatically detect your installed Python so we need to change it.
+When this happens Webots will print errors to its console and your robot will not move.
+
+You will need the full path to the version of Python that you want to use.
+This will vary based on the system you have.
+One way to find the path is by launching Python and running the following code:
+
+~~~~~ python
+import sys
+print(sys.executable)
+~~~~~
+
+Once you have the path you need to enter this into the Webots settings.
+
+To do this on Windows, open webots and go to the menu **Tools** &rarr; **Preferences** &rarr; **General** &rarr; **Python command** and enter the path in that box.
+Your Python path is likely similar to `C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python311\python.exe` when using Python 3.11, where `<USERNAME>` is your login.
+
+On Mac you can set the path to the Python version to use via **Webots** &rarr; **Preferences** <kbd>⌘</kbd><kbd>,</kbd>.
+Your Python path is likely similar to `/Library/Frameworks/Python.framework/Versions/3.11/bin/python3` when using Python 3.11.
```

Comment:
Thought this might be worth mentioning. Although people who use Homebrew would probably be able to figure this out. 

```suggestion
Your Python path is likely similar to `/Library/Frameworks/Python.framework/Versions/3.11/bin/python3` when using Python 3.11.
If you installed Python using Homebrew, it will be in `/opt/homebrew/bin/python3.11` (or `/usr/local/bin/python3.11` on an Intel-based Mac)
```